encoding/serialization.go

```diff
@@ -22,6 +23,41 @@ func (c *Frame) Deserialize(data []byte) (*Frame, error) {
 	return c, err
 }
 
+func (c *Frame) SerializeGnark() ([]byte, error) {
+	coded := make([]byte, 0, bn254.SizeOfG1AffineCompressed+BYTES_PER_SYMBOL*len(c.Coeffs))
```

Comment:
Do you want to consider to use SizeOfG1AffineUncompressed?  Marshal uses uncompressed format. https://github.com/consensys/gnark-crypto/blob/v0.6.1/ecc/bn254/marshal.go#L607

In current EigenDA, we use uncompressed format with 64 bytes

Reply:
We could settle on a 32 bytes point for this new encoding format